src/søknad/steg/4-barnasbosted/BarnetsBostedLagtTil.tsx

```diff
@@ -0,0 +1,129 @@
+              )}
+            </Element>
+            <Normaltekst>{forelder.navn}</Normaltekst>
+          </div>
+        )}
+        {forelder.fødselsdato && (
+          <div className="spørsmål-og-svar">
+            <Element>
+              {intl.formatMessage({ id: 'datovelger.fødselsdato' })}
+            </Element>
+            <Normaltekst>{formatDate(forelder.fødselsdato)}</Normaltekst>
+          </div>
+        )}
+        <div className="spørsmål-og-svar">
+          <Element>
+            {hentBeskjedMedNavn(
+              barn.navn,
+              intl.formatMessage({ id: 'barnasbosted.spm.andreForelderNorge' })
+            )}
+          </Element>
+          <Normaltekst>{forelder.borINorge ? 'Ja' : 'Nei'}</Normaltekst>
+        </div>
+        <div className="spørsmål-og-svar">
+          <Element>
+            {hentBeskjedMedNavn(
+              barn.navn,
+              intl.formatMessage({ id: 'barnasbosted.avtale' })
+            )}
+          </Element>
+          <Normaltekst>
+            {forelder.avtaleOmDeltBosted ? 'Ja' : 'Nei'}
+          </Normaltekst>
+        </div>
+        <div className="spørsmål-og-svar">
+          <Element>
+            {hentBeskjedMedNavn(
+              barn.navn,
+              intl.formatMessage({
+                id: 'barnasbosted.spm.harAnnenForelderSamværMedBarn',
+              })
+            )}
+          </Element>
+          <Normaltekst>{forelder.harAnnenForelderSamværMedBarn}</Normaltekst>
+        </div>
+        {forelder.harDereSkriftligSamværsavtale ? (
+          <div className="spørsmål-og-svar">
+            <Element>
+              {hentBeskjedMedNavn(
+                barn.navn,
+                intl.formatMessage({
+                  id: 'barnasbosted.spm.harDereSkriftligSamværsavtale',
+                })
+              )}
+            </Element>
+            <Normaltekst>{forelder.harDereSkriftligSamværsavtale}</Normaltekst>
+          </div>
+        ) : null}
+        {forelder.hvordanPraktiseresSamværet ? (
+          <div className="spørsmål-og-svar">
+            <Element>
+              {intl.formatMessage({ id: 'barnasbosted.element.samvær' })}
+            </Element>
+            <Normaltekst>{forelder.hvordanPraktiseresSamværet}</Normaltekst>
+          </div>
+        ) : null}
+        {forelder.borISammeHus ? (
+          <div className="spørsmål-og-svar">
+            <Element>
+              {hentBeskjedMedNavn(
+                barn.navn,
+                intl.formatMessage({
+                  id: 'barnasbosted.spm.borISammeHus',
+                })
+              )}
+            </Element>
+            <Normaltekst>{forelder.borISammeHus}</Normaltekst>
+          </div>
+        ) : null}
+        <div className="lenke-knapp" onClick={endreInformasjon}>
+          <img alt="Endre" src={endre} />
+          <Normaltekst>
+            {intl.formatMessage({ id: 'barnasbosted.knapp.endre' })}
+          </Normaltekst>
+        </div>
```

Comment:
Kunne denne trekkes ut til en egen komponent som tar inn onclick og tekstid som props? Ser for meg at denne kan gjenbrukes 🤩

Reply:
Og så kunne man trukket ut less-koden og brukt styled-components i samme fil ✨

Reply:
God idé, gjør det :)